app/helpers/sciruby_github_helper.rb

```diff
@@ -1,2 +1,26 @@
 module ScirubyGithubHelper
+  def export(dataframe)
+    respond_to do |format|
+      format.html
+
```

Comment:
@Shekharrajak - Could this `export` method be added to daru-view as a plugin gem when daru-io is also added? (optional dependency)